automatic/dbatools/dbatools.nuspec

```diff
@@ -1,74 +1,35 @@
 <?xml version="1.0" encoding="utf-8"?>
-<!-- Read this before creating packages: https://chocolatey.org/docs/create-packages -->

```

Comment:
Can you add back all the removed comments?

Reply:
Ah, I habitually follow the instructions in the Chocolatey package TODO - `- nuspec - fill this out, then clean out all the comments`. You'd prefer them left in, on purpose?